### ISSUE TITLE
deprecate / remove ncm-moab and ncm-maui

### ISSUE THREAD

**Issue by stdweird** [quattor/configuration-modules-grid#85]
* ncm-moab: https://github.com/quattor/configuration-modules-core/pull/633
* ncm-maui: looks like filecopy

**Comment by jouvin** [comment]
LAL workshop: `ncm-maui` is still used by the grid configuration... as it is some sort of legacy thing, it would be easier to keep it as is but could also be replaced by `metaconfig`MOAB component...

**Comment by jrha** [comment]
We can make it throw deprecation warnings without doing anything else, it should at least warn people that using it is a bad idea.

**Comment by jouvin** [comment]
Yes, we could do it... but at the same time it will not have any impact on the site using them who will not maintain the associated templates, as discussed at the workshop... So is it really worth the change?

**Comment by jrha** [comment]
Probably not, but it might put off anyone from using it who doesn't already use it... If they were crazy enough to consider it.

**Comment by jrha** [comment]
@jouvin I assume no-one is actually using this now and we can get rid of it?
